### PR TITLE
Fix #2570 : Fixed Typos and Errors

### LINKED ISSUE #2570: Fix typos and errors for the following files
1. In KeyboardHelper - there is a typo in `softinputkeybord` (3 changes)
     https://github.com/oppia/oppia-android/blob/74db288b9ac25fb942a8feb35e918bcd7e94e55c/app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt#L7
2. In StartSnapHelper - 'Throws' is not imported (1 change)
    https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt#L18
3. In StateAssemblerMarginBindingAdapters - typo in `split-veiw` (1 change)
     https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/databinding/StateAssemblerMarginBindingAdapters.java#L9

### PR DESCRIPTION
**Pull request by riturajjain2000**
## Explanation
Fixes #2570 
Fixed Typos and errors in 3 Files

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Review by rt4914 — CHANGES_REQUESTED** [review]
While we are fixing this lets replace `softinputkeyboard ` with `SoftInputKeyboard`.

**Comment by riturajjain2000** [comment]
> While we are fixing this lets replace `softinputkeyboard ` with `SoftInputKeyboard`.

Done!

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by riturajjain2000** [comment]
> LGTM, thanks.

Can this be merged ?

**Comment by MaskedCarrot** [comment]
@riturajjain2000 please wait for @BenHenning review on this.

**Comment by riturajjain2000** [comment]
> @riturajjain2000 please wait for @BenHenning review on this.

Ok sure.

File: app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt]: Why is this needed? It doesn't seem to be used. Suggest removing it.

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]: ``SoftInputKeyboard`` isn't a class. Suggest writing it out, instead: ``the soft input keyboard``. Ditto below.

**Review by BenHenning — APPROVED** [review]
Thanks @riturajjain2000! Had two comments. The change in my codeowner file LGTM, but @rt4914 will need to approve the other two files.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Ah it looks like @rt4914's approval was cleared, but it's still considered approved. Switching my review to 'request changes' for the open comments.

File: app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt
Comment (FareesHussain) [reply app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt]: @BenHenning Actually, I mentioned this in the issue. 
It is used in the below line not sure if this is needed 
https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt#L18

Comment (FareesHussain) [reply app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt]: @riturajjain2000 The PR reviewer can resolve the conversation after confirming the suggested changes are done or if anything left to discuss. The conversation should not be resolved by PR authors.

Comment (riturajjain2000) [reply app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt]: Ok ,I'll keep that in mind

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]:
```suggestion
     * This method hides the soft input keyboard.
     *
     * @param view the input view
     * @param context the activity context
```

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]:
```suggestion
     * This method shows the soft input keyboard.
     *
     * @param view the input view
     * @param context the activity context
```

**Review by BenHenning — APPROVED** [review]
Thanks @riturajjain2000. Added some opportunistic suggested fixes. Once addressed, this PR LGTM.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Actually, requesting changes to block submission until the last two comments are addressed (just to make sure it doesn't get accidentally merged).

**Comment by riturajjain2000** [comment]
> Actually, requesting changes to block submission until the last two comments are addressed (just to make sure it doesn't get accidentally merged).

Made changes as suggested

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
@riturajjain2000 I have restarted the tests, if they fail again, just update your branch with latest develop and it should work.

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]:
```suggestion
     * This method hides the soft input keyboard.
```

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]:
```suggestion
     * This method shows the soft input keyboard.
```

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @riturajjain2000! Couple of things:
1. Please make sure to fully read comments before considering them addressed (e.g. in this case you missed the '.' in my original suggested code). For suggested code changes, it's also fine just to commit that code directly to your branch.
2. Please always respond to comment threads saying you've addressed them, otherwise the assumption is that you might not of. We require all discussion threads to be addressed before a PR will be merged.

**Comment by riturajjain2000** [comment]
> Thanks @riturajjain2000! Couple of things:
> 
> 1. Please make sure to fully read comments before considering them addressed (e.g. in this case you missed the '.' in my original suggested code). For suggested code changes, it's also fine just to commit that code directly to your branch.
> 2. Please always respond to comment threads saying you've addressed them, otherwise the assumption is that you might not of. We require all discussion threads to be addressed before a PR will be merged.

OK , I'll keep this in mind from now on.

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (riturajjain2000) [reply app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]: > 

I addressed all the changes in a new PR. PTAL

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt
Comment (riturajjain2000) [reply app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt]: I addressed all the changes in a new PR. PTAL

**Pull request closed without merging**
